tests/compiler/tast.nim

```diff
@@ -0,0 +1,46 @@
+import compiler/ast
```

Comment:
Instead of this module we should have an explicit lookup table from TNodeKind to macros.NodeKind. This is cleaner than the current solution and gives us compile-time checking.